tests/sentry/workflow_engine/processors/test_workflow.py

```diff
@@ -97,16 +107,20 @@ def setUp(self):
         self.job = WorkflowJob({"event": self.group_event})
 
     def test_workflow_trigger(self):
-        triggered_workflows = evaluate_workflow_triggers({self.workflow}, self.job)
+        triggered_workflows, workflows_to_enqueue = evaluate_workflow_triggers(
+            {self.workflow}, self.job
+        )
         assert triggered_workflows == {self.workflow}
+        assert not workflows_to_enqueue
```

Comment:
is this not enqueued because it has a fast condition?

Reply:
yep, there's no slow condition and the workflow passes the fast condition